src/compute/src/compute_state.rs

```diff
@@ -1356,6 +1381,8 @@ struct ReportedFrontiers {
     write_frontier: ReportedFrontier,
     /// The reported input frontier.
     input_frontier: ReportedFrontier,
+    /// The reported output frontier.
+    output_frontier: ReportedFrontier,
```

Comment:
Maybe mention that this can be different from the `write_frontier` for e.g. REFRESH MVs.